test/spec.yml

```diff
@@ -884,7 +925,7 @@
       - npm i shelljs
       - |-
         cat <<EOF >/home/dockeruser/test/function.js
-        
+
```

Comment:
It's better to separate white space fixes and non related changes from PRs that change or add functionality.

Reply:
agree